bin/MadGraph5_aMCatNLO/cards/production/13TeV/HToAATo4B/SUSY_VBFH_HToAATo4B_M-MASS/SUSY_VBFH_HToAATo4B_M-MASS_run_card.dat

```diff
@@ -0,0 +1,270 @@
+# Tag name for the run (one word)                                    *
+#*********************************************************************
+  tag_1     = run_tag ! name of the run 
+#*********************************************************************
+# Run to generate the grid pack                                      *
+#*********************************************************************
+  .false.     = gridpack  !True = setting up the grid pack
+#*********************************************************************
+# Number of events and rnd seed                                      *
+# Warning: Do not generate more than 1M events in a single run       *
+# If you want to run Pythia, avoid more than 50k events in a run.    *
+#*********************************************************************
+  1000 = nevents ! Number of unweighted events requested 
+      0       = iseed   ! rnd seed (0=assigned automatically=default))
+#*********************************************************************
+# Collider type and energy                                           *
+# lpp: 0=No PDF, 1=proton, -1=antiproton, 2=photon from proton,      *
+#                                         3=photon from electron     *
+#*********************************************************************
+        1     = lpp1    ! beam 1 type 
+        1     = lpp2    ! beam 2 type
+     6500     = ebeam1  ! beam 1 total energy in GeV
+     6500     = ebeam2  ! beam 2 total energy in GeV
+#*********************************************************************
+# Beam polarization from -100 (left-handed) to 100 (right-handed)    *
+#*********************************************************************
+        0     = polbeam1 ! beam polarization for beam 1
+        0     = polbeam2 ! beam polarization for beam 2
+#*********************************************************************
+# PDF CHOICE: this automatically fixes also alpha_s and its evol.    *
+#*********************************************************************
+ lhapdf            = pdlabel     ! PDF set                                     
+ $DEFAULT_PDF_SETS = lhaid
+#*********************************************************************
+# Renormalization and factorization scales                           *
+#*********************************************************************
+ F        = fixed_ren_scale  ! if .true. use fixed ren scale
+ F        = fixed_fac_scale  ! if .true. use fixed fac scale
+ 91.1880  = scale            ! fixed ren scale
+ 91.1880  = dsqrt_q2fact1    ! fixed fact scale for pdf1
+ 91.1880  = dsqrt_q2fact2    ! fixed fact scale for pdf2
+ 1        = scalefact        ! scale factor for event-by-event scales
+#*********************************************************************
+# Matching - Warning! ickkw > 1 is still beta
+#*********************************************************************
+ 1        = ickkw            ! 0 no matching, 1 MLM, 2 CKKW matching
```

Comment:
You don't need to turn on matchings. You don't have multipartons.
it is only used for the case where you do
p p > X
p p > X j

Reply:
Hi @sihyunjeon ,
Does this line where I have:
p p > h01 j j
not count as "multipartons"?  Is it only if I have two different lines, with different numbers of extra jets "j"?
https://github.com/abrinke1/genproductions/blob/HToAATo4B_Cards_Run2_UL_VBFH/bin/MadGraph5_aMCatNLO/cards/production/13TeV/HToAATo4B/SUSY_VBFH_HToAATo4B_M-MASS/SUSY_VBFH_HToAATo4B_M-MASS_proc_card.dat#L5

Reply:
Definition of multiparton is when you need jet matching.
So "p p > X j j" alone is not multiparton sample
if you also have "p p > X j j j", it is